src/pages/guest-chat/ui/page.tsx

```diff
@@ -3,6 +3,8 @@ import { redirect } from 'next/navigation';
 import { WebsocketProvider } from '@/features/websocket';
 import { ROUTE } from '@/shared/config';
 import { Chatting } from '@/widgets/chatting';
+import ChattingHeader from '@/widgets/chatting/ui/chatting-header';
+import ChattingTitle from '@/widgets/chatting/ui/chatting-title';
```

Comment:
```suggestion
import ChattingHeader from '@/widgets/chatting';
import ChattingTitle from '@/widgets/chatting';
```
FSD 구조에 맞게 `@/widgets/chatting/index.ts`에 해당 컴포넌트 export 해서 사용해주세요!